packages/smart-wallet/src/smartWallet.js

```diff
@@ -366,6 +428,87 @@ export const prepareSmartWallet = (baggage, shared) => {
             },
           });
         },
+
+        provideBrandToPurses() {
+          const brandToPurses = provideLazy(
+            walletPurses,
+            this.facets.helper,
```

Comment:
is this our idiom for "the key is this Exo"? is `helper` identity stable across restarts? and `this` isn't?

Reply:
`this` is indeed not a stable identity. The idiom is to use one of the facets. Oh... `self` seems more natural.